src/services/services.ts

```diff
@@ -376,7 +376,7 @@ namespace ts {
             super(pos, end);
         }
 
-        get unescapedText(): string {
+        get text(): string {
             return unescapeLeadingUnderscores(this.escapedText);

```

Comment:
Could that wait for another PR? I would want to see the performance difference between testing for `_` every time we call `.text` vs adding an extra field to every `Identifier`.